Client/src/Components/HttpStatusLabel/index.jsx

```diff
@@ -0,0 +1,65 @@
+import PropTypes from "prop-types";
+import { useTheme } from "@mui/material";
+import { BaseLabel } from "../Label";
+
+/**
+ * @component
+ * @param {Object} props
+ * @param {number | 'N/A'} props.status - The http status for the label
+ * @param {Styles} props.customStyles - CSS Styles passed from parent component
+ * @returns {JSX.Element}
+ * @example
+ * // Render a http status label
+ * <HttpStatusLabel status={404} />
+ */
+
+const getRoundedStatusCode = (status) => {
+	if (typeof status === "number" && status >= 100 && status < 600) {
+		return Math.floor(status / 100) * 100;
+	}
+	return status;
+};
```

Comment:
This comment continues the one made in IncidentTable. We are going to deal with different scenarios here. if we don't have a number between 100 and 600, we are going to return  "default" (I suggest changing N/A to default).

Reply:
As an addendum to this comment, I would say that type checking here isn't necessary, we should guarantee that this function only receives a number upstream anyways.

This function also has a mixed return type which is dangerous.  It will either return a `number` or `undefined`.

JavaScript's type coercion allows us to get away with this, but now we must always do a check whenever we use this function upstream to see if we have a value or not.

As @marcelluscaio mentioned we can avoid this by returning a default value.

```
if(status >= 100 && status < 600){
   // return roudned status code
}
return defaultCode
```

I'd say that is a safer and more reliable function.  Otherwise we must always do

```
if ( getRoundedStatusCode(status) !== undefined ) {
   // do stuff here
}
```

or at the very least something like 

```
const roundedStatusCode = getRoundedStatusCode(status) || 5000
```

whenever we use the function.  This adds extra verbosity and complexity to the code which is easily avoided.

 One of the reasons I'm an advocate for strongly typed languages.. or TypeScript at least :joy: